artifacts/AdditionalSetupArtifacts.ps1

```diff
@@ -379,16 +380,6 @@ $excludeAppsFromSaaSBak = @(
         Id     = 'f8e2d23f-5500-4b0a-b139-8a7b9725334e';
         Reason = "deprecated"
     },
-    [pscustomobject]@{
```

Comment:
Why remove this apps from list? There where some troubles in Containers because this apps only work in SaaS environments

Reply:
Accepted as dependency of OPplus